doc/development/developing-sql-dialect/implementing_additional_dialect_specific_behavior.md

```diff
@@ -198,8 +198,11 @@ Let's look at a HIVE dialect example. We only want to change mapping for one dat
     
         @Override
         public DataType mapJdbcType(final JdbcTypeDescription jdbcTypeDescription) {
-            if (jdbcTypeDescription.getJdbcType() == Types.DECIMAL) {
+            final int jdbcType = jdbcTypeDescription.getJdbcType();
+            if (jdbcType == Types.DECIMAL) {
```

Comment:
I would prefer to use a case for more readability in code, but it's just a personal taste.